wagtail/admin/action_menu.py

```diff
@@ -265,7 +250,6 @@ def _get_base_page_action_menu_items():
     if BASE_PAGE_ACTION_MENU_ITEMS is None:
         BASE_PAGE_ACTION_MENU_ITEMS = [
             SaveDraftMenuItem(order=0),
-            DeleteMenuItem(order=10),
             UnpublishMenuItem(order=20),
```

Comment:
Not changing the ordering here as I think that would change the order of users' custom menu items.